Mathlib/Algebra/Star/Conjneg.lean

```diff
@@ -0,0 +1,75 @@
+/-
+Copyright (c) 2024 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.BigOperators.Pi
+import Mathlib.Algebra.Star.Pi
+import Mathlib.Data.Set.Pointwise.Basic
+
+/-!
+# Conjugation-negation operator
+
+This file defines the conjugation-negation operator, useful in Fourier analysis.
```

Comment:
Could be nice to additionally say here the way in which it's useful. eg, that convolution with conjneg f is adjoint to convolution with f